prettier.config.js

```diff
@@ -11,7 +11,6 @@ module.exports = {
             files: [
                 'src/content/**/*',
                 'src/tests/unit/tests/DetailsView/**/*',
-                'src/tests/unit/tests/electron/**/*',
```

Comment:
So to clarify, you aren't "adding the `src/tests/unit/tests/electron` folder to prettier override" but actually removing `src/tests/unit/tests/electron` folder from the prettier override? Or are the folders in the `files` array exempt from the prettier override, so removing the folder would make sure that the line width is in line with the prettier override changes we want to see?

Reply:
You're right: I'm removing the folder so it doesn't follow the override. Originally, when we started, we were operating the other way around: adding paths to the override, but at the point we added somewhere around 60% of the code base, Dan made the change, making everything follow the line width we want to have, and adding every path we were missing to the override. I being copy/pasting the description as the work is pretty much the same. 
Good catch.